src/pages/quickstarts.js

```diff
@@ -14,21 +13,19 @@ function HomepageHeader() {
     <header className={clsx('hero--primary', styles.heroBanner, styles.ali)}>
       <div className={clsx('container', styles.container)}>
         <div >
-          <Typography scale="headline1">Getting started</Typography>
+          <Typography scale="headline1">{translate({ message: 'developers.getting_started_title' })}</Typography>
           <br/>
           <p className="hero__subtitle">
-            <Translate id="theme.tagline">
-                Discover in-depth tutorials and guides about how to use Teradata Vantage from all members of the Teradata community, including employees, partners, customers.
-            </Translate>
+            {translate({ message: 'quickstarts.tagline' })}
           </p>
           <p className="hero__subtitle">
-            Existing customer or partner? Explore courses at <Link href='https://teradata.com/University/Overview' target='_blank'>Teradata University</Link>.
+          {translate({ message: 'quickstarts.tagline_2' })} <Link href='https://teradata.com/University/Overview' target='_blank'>Teradata University</Link>.
```

Comment:
We should also translate "Teradata University"

Reply:
Will do